contracts/mocks/Disposables/DisposableCover.sol

```diff
@@ -80,6 +80,31 @@ contract DisposableCover is MasterAwareV2, ReentrancyGuard {
     coverAssetsFallback = _coverAssetsFallback;
   }
 
+   /**
+   * @param paramNames  An array of elements from UintParams enum
+     * @param values An array of the new values, each one corresponding to the parameter
+   */
+  function updateUintParameters(
+    CoverUintParams[] calldata paramNames,
+    uint[] calldata values
+  ) external {
+
+    for (uint i = 0; i < paramNames.length; i++) {
+      if (paramNames[i] == CoverUintParams.globalCapacityRatio) {
+        globalCapacityRatio = uint24(values[i]);
+        continue;
+      }
+      if (paramNames[i] == CoverUintParams.globalRewardsRatio) {
+        globalRewardsRatio = uint24(values[i]);
+        continue;
+      }
+      if (paramNames[i] == CoverUintParams.coverAssetsFallback) {
+        coverAssetsFallback = uint32(values[i]);
+        continue;
+      }
+    }
+  }
+
```

Comment:
After reducing the size of the Cover contract and looking at the functions that the disposable contract adds, I think we can actually implement the disposable properly and it won't hit the size limit.

Quickly reviewing it, seems like the only significant differences between the two are:
1. The `setCoverAssetsFallback` function which is a one-liner
2. A require that checks the price to be higher than the `GLOBAL_MIN_PRICE_RATIO`

Let's switch to the version that implements the Cover contract and adds the `setCoverAssetsFallback` function.